flags/conditions.py

```diff
@@ -1,4 +1,5 @@
 import re
+from distutils.util import strtobool
```

Comment:
I'm curious, how does this pass the linter? Aren't these both considered `STDLIB` and thus their order should be swapped?

Reply:
Because all `import`s within a group (`STDLIB`) come before all `from` imports in that same group.

Reply:
Ah, right. For some reason I thought I'd seen this the other way as well. I do prefer doing it the way you have here.